controllers/matryoshka/internal/kubeapiserver/kubeapiserver.go

```diff
@@ -256,6 +256,10 @@ func (r *Resolver) apiServerVolumeMounts(server *matryoshkav1alpha1.KubeAPIServe
 }
 
 func (r *Resolver) apiServerCommand(server *matryoshkav1alpha1.KubeAPIServer) []string {
+	var featureGates []string
```

Comment:
Since we know the size of `server.Spec.FeatureGates` we can allocated the array with the correct size.